src/reconcile/rec_col.c

```diff
@@ -972,17 +963,10 @@ __wt_rec_col_var(
                       (!deleted && !last.deleted && size != 0 && last.value->size == size &&
                         memcmp(last.value->data, data, size) == 0))) {
                     /*
-                     * The start time pair for deleted keys must be (WT_TS_NONE, WT_TXN_NONE) and
-                     * stop time pair must be (WT_TS_MAX, WT_TXN_MAX) since we no longer select
-                     * tombstone to write to disk and the deletion of the keys must be globally
-                     * visible.
+                     * The time window for deleted keys must be empty.
                      */
-                    WT_ASSERT(session,
-                      (!deleted && !last.deleted) ||
-                        (last.tw.durable_start_ts == tw.durable_start_ts &&
-                          last.tw.start_ts == WT_TS_NONE && last.tw.start_txn == WT_TXN_NONE &&
-                          last.tw.durable_stop_ts == tw.durable_stop_ts &&
-                          !WT_TIME_WINDOW_HAS_STOP(&last.tw)));
```

Comment:
Did you make this change deliberately? The old check seemed a bit odd to me, but it was comparing values between `last` and `tw` - which I'd presumed were deliberate.

Reply:
I'm not sure why the code was wrote like that before but we already compare the two in line 970 so I think the change doesn't change the behavior.

Reply:
Got it - thanks!